docs/index.md

```diff
@@ -79,6 +79,20 @@ Add directives to the `*.scala` file:
 * Start tracing your application with [Jaeger and Docker](examples/jaeger-docker/README.md)
 * Implement tracing and metrics with [Honeycomb](examples/honeycomb/README.md)
 
+## The Noop Tracer  
+
+If you use a library that supports otel4s (eg [Skunk](https://github.com/typelevel/skunk)) but do not want to use Open Telemetry, then you can place the [No-op Tracer](https://www.javadoc.io/doc/org.typelevel/otel4s-docs_2.13/latest/org/typelevel/otel4s/trace/Tracer$.html) into implicit scope.
+
+```scala mdoc:silent
+// via import
+// import org.typelevel.otel4s.trace.Tracer.Implicits.noop
+
+// via implicit val
+import cats.effect.IO
+import org.typelevel.otel4s.trace.Tracer
```

Comment:
would it be better to suggest `import org.typelevel.otel4s.trace.Tracer.Implicits.noop`?

Reply:
It does (line 87):
```scala
// via import
// import org.typelevel.otel4s.trace.Tracer.Implicits.noop
```

Unfortunately, if the import is uncommented, there is a warning regarding 'unused' import.

Reply:
Hm, perhaps we should provide a better example to make it work with uncommented import. 

And I guess `Implicts.noop` should be a preferred option